_posts/hyotester/2022-09-22-hyotester-first-focused-reflection.md

```diff
@@ -1,8 +1,202 @@
 ---
 layout: post
 author: hyotester
-title: "Hyo Tester's First Focused Reflecction"
+title: "Hyo's First Focused Reflection"
 ---
 
-This is my first post!
+# Introduction
+Hi, I'm hyojeong! This posting is my first focused reflection, where I am exploring and focusing on code and how that specific code has helped me learn throughout the class. 
+
+## Let's get started to understand Functional Turtles
+
+September is my birthday!! So I tried to learn how to write the Happy Birthday Python Program during Turtle. Also, I use input function to know that How many friends will come to my birthday party?. 
+
+You may be wondering that who invited to my birthday party. So, I used function to iterate variable name inside loop. This function let you know the people  invited to my birthday party.
+
+<iframe src="https://trinket.io/embed/python/4103681aad" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+#The loop / variable name 
+
+```python
+for name in ["Joe", "Amy", "Brad", "Angelina", "Zuki", "Thandi", "Paris"]:
+    print("Hi", name, "Please come to my party on Saturday!")
+        
+```
+
+# Input 
```

Comment:
I might rename this to something like "User Input to Calculate Birth Year and Number of Friends" ... I think your code is really cool, but I think a more specific title would clarify what exactly your code is doing (this goes with what prof. Elliott was saying about writing for the humans reading the code) ... what do you think?

Reply:
Yes, exactly I was looking for a more specific title, and "User Input to Calculate Birth Year and Number of Friends" make sense to me! Thank you for the suggestion!